src/System.Diagnostics.Contracts/src/System.Diagnostics.Contracts.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <AssemblyName>System.Diagnostics.Contracts</AssemblyName>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
-    <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.0</NuGetTargetMoniker>
+    <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.7</NuGetTargetMoniker>
```

Comment:
whenever you bump this you will need to minimally update the associated project.json version otherwise you will see a bunch of warnings about them not finding this version. I also expect that you will likely need to make some updates to the pkgproj. @ericstj if we don't change assembly version but just bump the standard version shouldn't that hit some package validation issues?